pywwt/core.py

```diff
@@ -374,8 +378,8 @@ def _on_app_message_received(self, payload):
         if callback:
             try:
                 callback(self, updated_fields)
-            except Exception as e: # TODO: Something better here
-                print("Error when calling selection updated callback:\n%s" % str(e))
+            except Exception as e:
+                raise CallbackError() from e
```

Comment:
Hmmm. The thing to worry about here is that this function can be called from weird async Python event handlers where any exceptions might just disappear into the ether. I think the least-bad thing to do here is emulate what's done in `qt.py`, and use `from .logger import logger` paired with `logger.exception` to report any callback errors.